src/HttpServer/HttpServer.cc

```diff
@@ -25,6 +25,12 @@ using json = nlohmann::json;
 namespace carta {
 
 const std::string success_string = json({{"success", true}}).dump();
```

Comment:
My mistake; I didn't see that this was already a global variable and you just capitalised the name. In my original comment I suggested that this value should be used directly inside the functions -- so instead of `res->end(success_string)` we would use `res->end(json({{"success", true}}).dump())`. But now that I write that out, I think that it looks cleaner to keep this as a const variable, and you were correct to capitalise it (that's a more consistent style for constants). I apologise for the confusing feedback! Please restore the capitalisation.